src/App.js

```diff
@@ -85,12 +93,36 @@ export const App = () => {
     }
   };
 
+  const onConfirmBatchTransaction = preparedTransactions => {
+    setPreparedTransactions(preparedTransactions);
+    setTransactionStatus(TransactionStatus.pending);
+    history.replace('/transaction');
+  };
+
   const onConfirmTransaction = preparedTransaction => {
     setPreparedTransaction(preparedTransaction);
     setTransactionStatus(TransactionStatus.pending);
     history.replace('/transaction');
   };
 
+  const submitBatchTransactions = async preparedTransactions => {
+    try {
+      const { network, transactions, destinationData } = preparedTransactions;
+
+      const queue = transactions.map(transaction => sendTransaction(network, transaction));
+
+      const responses = await Promise.all(queue);
+
+      const transactionData = processTransactionResponses(network, responses, destinationData);
+
+      setTransactionData(transactionData);
+      setTransactionStatus(TransactionStatus.success);
+    } catch (error) {
+      console.error(error);
```

Comment:
I think console.error accepts only string e.g. we can do

console.error(error?.message)

Reply:
Good catch! 

Fixed: [c571671](https://github.com/uphold/vault-assist-tool/pull/27/commits/c571671042dc7628c40bb9e12cac9921e93509ce)